tests/e2e/main_test.go

```diff
@@ -77,3 +84,110 @@ func TestLintMetrics(t *testing.T) {
 		t.Fatalf("the problems encountered in Lint are: %v", problems)
 	}
 }
+
+func TestDocumentation(t *testing.T) {
+	labelsDocumentation, err := getLabelsDocumentation()
+	if err != nil {
+		t.Fatal("Cannot get labels documentation", err)
+	}
+
+	buf := &bytes.Buffer{}
```

Comment:
Not a strong opinion. But this is just my preference in the interest of semantics.

```suggestion
	var buf *bytes.Buffer
```